mkt/feed/views.py

```diff
@@ -811,6 +815,35 @@ def get_es_feed_element_query(self, sq, feed_items):
 
         return sq.filter(es_filter.Bool(should=filters))[0:len(feed_items)]
 
+    def _featured_website_query(self, region=None, n=11):
+        """
+        Query for n featured websites matching an optional region.
+        """
+        TAG = 'featured-website'
+        if region:
+            TAG = '%s-%s' % (TAG, region.slug)
+        daily_seed = int(datetime.now().strftime('%Y%m%d'))
+        mow_query = query.Q(
+            'function_score',
+            filter=es_filter.Bool(should=[es_filter.Term(tags=TAG)]),
```

Comment:
Yeah, good call. This was the remnants of me trying to be getting it into one too-cute query.